python/test/unit/parallel-assembly-solve/test_solve_result_against_reference.py

```diff
@@ -97,22 +100,10 @@ def print_errors(errors):
 
 def test_computed_norms_against_references():
     # Reference values for norm of solution vector
-    reference = {("16x16 unit tri square", 1): 9.547454087328376,
-                 ("16x16 unit tri square", 2): 18.42366670418269,
-                 ("16x16 unit tri square", 3): 27.29583104732836,
-                 ("16x16 unit tri square", 4): 36.16867128121694,
-                 ("4x4x4 unit tet cube", 1): 12.23389289626038,
-                 ("4x4x4 unit tet cube", 2): 28.96491629163837,
-                 ("4x4x4 unit tet cube", 3): 49.97350551329799,
-                 ("4x4x4 unit tet cube", 4): 74.49938266409099,
-                 ("16x16 unit quad square", 1): 9.550848071820747,
-                 ("16x16 unit quad square", 2): 18.423668706176354,
-                 ("16x16 unit quad square", 3): 27.295831017251672,
-                 ("16x16 unit quad square", 4): 36.168671281610855,
-                 ("4x4x4 unit hex cube", 1): 12.151954087339782,
-                 ("4x4x4 unit hex cube", 2): 28.965646690046885,
-                 ("4x4x4 unit hex cube", 3): 49.97349423895635,
-                 ("4x4x4 unit hex cube", 4): 74.49938136593539}
+    reference = {("16x16 unit tri square", 1): 9.544967937540488,
```

Comment:
Why fewer reference values than before?

Reply:
Were not used. So I just deleted. And with UFL expression it gives slightly different numbers anyway.